streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -88,6 +88,58 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
     }
   }
 
+  test("SPARK-6766: batch info should be submitted") {
```

Comment:
[info] StreamingListenerSuite:
[info] - batch info reporting (4 seconds, 976 milliseconds)
[info] - receiver info reporting (847 milliseconds)
[info] - SPARK-6766: batch info should be submitted (667 milliseconds)
[info] - SPARK-6766: processingStartTime of batch info should not be None when starting (691 milliseconds)
[info] ScalaTest
[info] Run completed in 8 seconds, 996 milliseconds.
[info] Total number of tests run: 4
[info] Suites: completed 1, aborted 0
[info] Tests: succeeded 4, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.